build/WindowsAppSDK-Versions.yml

```diff
@@ -1,6 +1,6 @@
 variables:
   # Version Control
-  major: "0"
-  minor: "8"
+  major: "1"
+  minor: "0"
   patch: "0"
-  branch: 'main'
\ No newline at end of file
+  branch: 'exp'
```

Comment:
Should this go directly into the experimental branch?

Reply:
oh whoops my bad. It's meant to be on the experimental branch